editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -314,8 +314,10 @@ export const NavigatorItem: React.FunctionComponent<NavigatorItemInnerProps> = b
       [dispatch, elementPath, selected, isHighlighted],
     )
     const focusComponent = React.useCallback(() => {
-      dispatch([EditorActions.setFocusedElement(elementPath)])
-    }, [dispatch, elementPath])
+      if (isFocusedComponent) {
```

Comment:
Shouldn't this be `if (isFocusableComponent) {`?

Reply:
Why yes, yes it should, fixed in 650a8f4e.